actions/views/rhs.ts

```diff
@@ -251,6 +281,46 @@ export function showPinnedPosts(channelId?: string) {
     };
 }
 
+export function showChannelFiles(channelId?: string) {
+    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
+        const state = getState();
+        const currentChannelId = getCurrentChannelId(state);
+        const teamId = getCurrentTeamId(state);
+
+        dispatch(batchActions([
```

Comment:
Can we eliminate `batchActions` here given there's just a single one?

Reply:
Done